matrix/lambdas/daemons/worker.py

```diff
@@ -37,13 +39,28 @@ def run(self, format: str, worker_chunk_spec: dict):
         """
         # TO DO pass in the parameters in worker chunk spec flat
         self._parse_worker_chunk_spec(worker_chunk_spec)
-        dss_zarr_store = DSSZarrStore(bundle_uuid=self._bundle_uuid,
-                                      bundle_version=self._bundle_version,
-                                      dss_instance=self._deployment_stage)
-        group = zarr.group(store=dss_zarr_store)
-        exp_df, qc_df = convert_dss_zarr_root_to_subset_pandas_dfs(group, self._input_start_row, self._input_end_row)
+        exp_dfs = []
+        qc_dfs = []
+        for chunk_idx in range(len(self._bundle_uuids)):
+            dss_zarr_store = DSSZarrStore(bundle_uuid=self._bundle_uuids[chunk_idx],
+                                          bundle_version=self._bundle_versions[chunk_idx],
+                                          dss_instance=self._deployment_stage)
+            group = zarr.group(store=dss_zarr_store)
```

Comment:
I know you didn't write this code, but I think this should be a responsibility of `convert_dss_zar_root_to_subset_pandas_dfs` and that function should accept a `dss_zarr_store` as opposed to a zarr group (actually the zarr group is available via `dss_zarr_store`)